### PR TITLE
py sol for small multiple

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC: https://atcoder.jp/contests/arc084/submissions/58326501

File: solutions/gold/ac-smallmultiple.mdx [hunk 1]
```diff
@@ -91,4 +91,40 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+from collections import deque
+
+INF = 10**9 + 7
```
Comment (SansPapyrus683) [inline solutions/gold/ac-smallmultiple.mdx:97]: float("inf")

Comment (freakin23) [reply solutions/gold/ac-smallmultiple.mdx:97]: okay

**Comment by freakin23** [comment]
upd AC: https://atcoder.jp/contests/arc084/submissions/58326989

File: solutions/gold/ac-smallmultiple.mdx [hunk 2]
```diff
@@ -90,5 +89,40 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+from collections import deque
+
+INF = float("inf")
```
Comment (SansPapyrus683) [inline solutions/gold/ac-smallmultiple.mdx:97]: no need for the constant now so just replace it

Comment (freakin23) [reply solutions/gold/ac-smallmultiple.mdx]: done

**Pull request merged**